examples/blaze/examples-blaze.sbt

```diff
@@ -6,6 +6,10 @@ publishArtifact := false
 
 fork := true
 
+libraryDependencies ++= Seq(
+  "io.dropwizard.metrics" % "metrics-json" % "3.1.0"
```

Comment:
Let's use the definition in Http4sBuild.  That centralizes our dependency declarations and keeps them in sync.